src/geometry/GeoParams.cc

```diff
@@ -125,5 +139,19 @@ GeoParamsPointers GeoParams::device_pointers() const
     return result;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Increase CUDA stack size to enable complex geometries.
+ *
+ * For example, CMS requires a stack limit of at least 8192 * 4.
```

Comment:
The actual requirement could be less, I did not test.  From what I saw 8192 * 2 might work.

Reply:
OK, so maybe change wording to:
> For the `cms2018.gdml` detector geometry, the default stack size is too small, and a limit of 32768 is recommended.
?